tests/ai_unet/test_model_training.py

```
@@ -0,0 +1,364 @@
+            orig_img_b = model_training.IMAGES_DIR_B
+            orig_mask_b = model_training.MASKS_DIR_B
+
+            # 1. Test standard combine success
+            model_training.IMAGES_DIR_A = img_dir
+            model_training.MASKS_DIR_A = mask_dir
+            model_training.IMAGES_DIR_B = img_dir
+            model_training.MASKS_DIR_B = mask_dir
+
+            train_tf = model_training.get_train_transforms()
+            val_tf = model_training.get_val_transforms()
+            train_ds, val_ds = model_training.build_datasets(train_tf, val_tf)
```

Ruff (0.15.12)

[warning] Unpacked variable `val_ds` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)